docs/pages/reference/join-methods.mdx

```diff
@@ -0,0 +1,427 @@
+---
+title: Join Methods and Token Reference
+description: Describes the different ways to configure a node to join the Teleport cluster.
+keywords: [join, joining, token]
+---
+
+<Admonition type="note" title="Prerequisite">
+You must be familiar with the [Teleport Core concepts](../core-concepts.mdx) before reading this page.
+</Admonition>
+
+## Definitions
+
+### Joining
+
+Joining a Teleport cluster is the act of establishing trust between a new node and
+all the existing nodes, already part of the Teleport cluster. This trust is
+established via certificates. Once a node has its certificates, it can interact
```

Comment:
Trust is not established via certificates, as you don't get a host certificate until after we've established trust (via a shared token or workload identity).